### ISSUE TITLE
delay failed on ajax call

### ISSUE THREAD

**Issue by Hongbo-Miao** [ReactiveX/rxjs#3133]
I am trying to delay the ajax call.

**RxJS version:** 5.52

**Code to reproduce:**
```js
ajax.get(url).delay(1000);
```

**Expected behavior:**
Delay 1s

**Actual behavior:**
No delay

**Additional information:**
I found the walkaround way at [here](https://github.com/Reactive-Extensions/RxJS/issues/1420) by

```js
Observable.timer(1000).flatMapLatest(ajax.get(url))
```

But it would be great to fix it.

**Comment by ptitjes** [comment]
I believe that what is intended when you do `ajax.get(url).delay(1000)` is to delay the propagation of the **results** of the ajax request.

**Comment by Brooooooklyn** [comment]
you should use:

```ts
import { async } from 'rxjs/scheduler/async'

ajax.get(url).subscribeOn(async, 1000)
```

**Comment by Hongbo-Miao** [comment]
Thanks, @Brooooooklyn . I tried, it also works. Would you mind explaining a little bit more why choose this? Thanks

**Comment by Brooooooklyn** [comment]
in this case 
```ts
const a$ = ajax.get(url).delay(1000)
```
when you called `a$.subscribe` , ajax would execute immediately.
`delay` operator here is just take the response of ajax and delay to pass it to `next` `operator`.

`subscribeOn ` take a scheduler, which means it would change your `subscribe time scheduler`.

it just equal to:

```ts
setTimeout(() => {
  $a.subscribe()
}, 1000)
```

remember, whenever you call `subscribe` on a `Observable`, the `source` of this `Observable` would execute immediately, only `subscribeOn` operator could change it.

**Comment by benlesh** [comment]
This isn't a bug, so I'm going to close. The workaround is what you wanted to do. :)
